### PR TITLE
Sprint action staging

### LINKED ISSUE #185: Github Actions for LibraryServiceStatusSystem UI
All of these will include badges:

- [x] Build
- [x] Test Coverage
- [x] Ensure PR checks are active

Where it is possible we should reuse workflows between repos.

### PR DESCRIPTION
**Pull request by rmathew1011**
Resolves #185

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        uses: actions/setup-node@v2
+        with:
+          node-version: 8
+
+      - name: "Install Node"
+        run: npm install
+
+      - name: "Starting Service"
+        run: |
+          npm start > /dev/null &
```
Comment (wwelling) [inline .github/workflows/build.yml:34]: I think we should make this consistent to use xvfb to run chrome browser.

**Review by wwelling — APPROVED** [review]
Maybe make consistent use of xvfb for headless browser future work. This is good to migrate off of travis.

File: .github/workflows/build.yml (hunk 1 above)
Comment (rmathew1011) [reply .github/workflows/build.yml:34]: Utilized xvfb to run chrome browser.

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+      - name: "Running Tests"
+        run: npm run test-ci
+
+      - name: "Coverage Report"
+        uses: MikeEdgar/github-action@raw_coverage_file
+        with:
+          github-token: ${{ secrets.github_token }}
+          path-to-file: './coverage/coveralls.json'
+          coverage-format: raw
```
Comment (wwelling) [inline .github/workflows/build.yml:43]: Asking for a newline at end of the file.

**Review by wwelling — APPROVED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (rmathew1011) [reply .github/workflows/build.yml:43]: Added line at end of file.

**Review by wwelling — APPROVED** [review]

**Pull request merged**